src/words/Productive.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+
+import LoadingAnimation from "../components/LoadingAnimation";
+import { setTitle } from "../assorted/setTitle";
+import strings from "../i18n/definitions";
+import Word from "./Word";
+import * as s from "../components/TopMessage.sc";
+import { UMR_SOURCE } from "../reader/ArticleReader";
+
+export default function Productive ({ api }) {
+  const [words, setWords] = useState(null);
+  const [productiveExercisesEnabled, setProductiveExercisesEnabled] = useState();
+
+  useEffect(() => {
+    api.topBookmarks(300, (topWords) => {
```

Comment:
the productive should not be filtered from `topBookmarks` but rather from `bookmarksToStudy` I think.

Reply:
This should reflect the words the user is learning, with the changes I am implementing bookmarksToStudy should only have the boomarks the student is learning, (without scheduling them) - currently this behaviour would make the system add new bookmarks every time call it up to that limit. So maybe TODO for when I merge the notification changes with the new endpoints?